package.json

```diff
@@ -62,5 +65,8 @@
     "validate-commit-msg": "^2.14.0",
     "webpack": "^3.11.0",
     "webpack-dev-server": "^2.11.2"
+  },
+  "dependencies": {
+    "react-router-dom": "^4.2.2"
```

Comment:
this should be a dev dependency too. None of our shipped NPM modules will depend on react-router...just our developer workflow